ui/client/components/modals/ValidationLabels.tsx

```diff
@@ -1,25 +1,49 @@
 import React from "react"
-import {v4 as uuid4} from "uuid"
+import _ from "lodash"
```

Comment:
I think we try not to import _ anymore, but only what is needed

Reply:
👍

Reply:
@mproch 
https://github.com/TouK/nussknacker/pull/1191/commits/1c0a0ce863576402dfc9259886c0786b62629dee